### PR TITLE
Implementation of led config & PCA9685

### PR DESCRIPTION
**Pull request by kretowicz14**
Remember to install libs form Readme and update pip dependencies.
We need also update docs about update boneIO on docs page

### CODE REVIEW

File: boneio/relay/pca.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+    @property
+    def last_brightness(self) -> int:
+        """Get brightness in 0-255 scale."""
+        return self._last_brightness
+
+    @last_brightness.setter
+    def last_brightness(self, value: int):
+        """Set brightness in 0-255 vale, and convert to hex for PCA."""
+        self._last_brightness = value
```
Comment (kretowicz14) [inline boneio/relay/pca.py:112]: Need to fix comments to value range 0-65535

File: boneio/relay/pca.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    def state(self) -> str:
+        """Is relay active."""
+        return self._state
+
+    @property
+    def brightness(self) -> int:
+        """Get brightness in 0-255 scale."""
+        return self._pin.duty_cycle
+
+    @brightness.setter
```
Comment (pszafer) [inline boneio/relay/pca.py:96]: setter can only be used for simple set, if send_state is going to be call better to use `set_brightness` I think.
According to pep:
```
Note 2: Avoid using properties for computationally expensive operations; the attribute notation makes the caller believe that access is (relatively) cheap.
```

File: boneio/relay/pca.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    def brightness(self, value: int):
+        """Set brightness in 0-255 vale, and convert to hex for PCA."""
+        _LOGGER.debug("Set brightness relay.")
+        self._pin.duty_cycle = value
+        self.last_brightness = value
+        self.send_state()
+
+    @property
+    def last_brightness(self) -> int:
```
Comment (pszafer) [inline boneio/relay/pca.py:105]: What's the purpose of the last brigthness?

**Review by pszafer — COMMENTED** [review]
Questions about brightness

File: boneio/relay/pca.py (hunk 2 above)
Comment (kretowicz14) [reply boneio/relay/pca.py]: Fixed

**Pull request merged**